README.md

```diff
@@ -32,6 +32,8 @@ This program has following dependencies:
 * Node.js
 * Yeoman
 
+Yoeman requires the Node to have a version higher than v14
```

Comment:
Change "Yoeman" to "Yeoman"

Reply:
Sorry, but Yoeman is already spelled correctly

Reply:
Sorry for my mistake, it went unnoticed. I just fixed it, thank you!